src/components/TopBar/CallButton.vue

```diff
@@ -328,13 +328,13 @@ export default {
 		},
 
 		handleClick() {
-			const shouldShowDeviceCheckerScreen = (BrowserStorage.getItem('showDeviceChecker' + this.token) === null
-				|| BrowserStorage.getItem('showDeviceChecker' + this.token) === 'true') && !this.forceJoinCall
-			console.debug(shouldShowDeviceCheckerScreen)
-			if (((this.isStartingRecording || this.isRecording) && !this.forceJoinCall) || shouldShowDeviceCheckerScreen) {
-				emit('talk:device-checker:show')
+			const shouldShowMediaSettingsScreen = (BrowserStorage.getItem('showMediaSettings' + this.token) === null
+				|| BrowserStorage.getItem('showMediaSettings' + this.token) === 'true') && !this.forceJoinCall
```

Comment:
**Must:** This resets basically the behaviour and all users that opted out of it will see it again. Is it needed?
```suggestion
			const shouldShowMediaSettingsScreen = (BrowserStorage.getItem('showDeviceChecker' + this.token) === null
				|| BrowserStorage.getItem('showDeviceChecker' + this.token) === 'true') && !this.forceJoinCall
```